scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -931,18 +931,10 @@ class FormatWriter(formatOps: FormatOps) {
             case t: Scaladoc.Tag =>
               sb.append(t.tag.tag)
               t.label.foreach(x => sb.append(' ').append(x.syntax))
-              t.desc match {
-                case Some(x: Scaladoc.Text) if t.tag.tag == "@usecase" =>
-                  // scaladoc parser doesn't allow newlines in @usecase
-                  val words = x.part.iterator.map(_.syntax)
-                  words.foreach(sb.append(' ').append(_))
-                  appendBreak()
-                case Some(x: Scaladoc.Text) =>
-                  appendBreak()
-                  val tagMargin = getIndentation(2 + termIndent.length)
-                  sb.append(tagMargin)
-                  formatTextAfterMargin(x, tagMargin)
-                case _ => appendBreak()
+              appendBreak()
```

Comment:
Will this not break for `@usecase` ?

Reply:
it will not, @dos65 and i changed tags with a label and no description to behave like usecase and take the entire line rather than single word.